babel.config.js

```diff
@@ -1,31 +0,0 @@
-// Only used for Jest testing
```

Comment:
this file is not necessary since switching to vitest.

Unfortunately we still do have a dependency on babel, but now its only from `@storybook/test-runner`